packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.types.ts

```diff
@@ -62,6 +64,9 @@ export interface IBreadcrumbProps extends React.Props<Breadcrumb> {
    * Optional index where overflow items will be collapsed. Defaults to 0.
    */
   overflowIndex?: number;
+
+  getStyles?: IStyleFunction<IBreadcrumbStyleProps, IBreadcrumbStyles>;
+  theme?: ITheme;
```

Comment:
I didn't think of this when I did my PR, but should we consider defining an IStyle interface and extend from it here? I think there are a few benefits: consistency, traceability, and docs benefits like @inheritdoc.

Reply:
yes probably, but maybe in a cleanup PR.